src/index.ts

```diff
@@ -196,7 +196,8 @@ prog
 
     const installSpinner = ora(Messages.installing(deps)).start();
     try {
-      await execa(getInstallArgs(getInstallCmd(), deps));
+      const installArgs = getInstallArgs(getInstallCmd(), deps);
+      await execa(installArgs[0], installArgs.slice(1));
```

Comment:
The typings of `execa` say you can't pass an array of strings as the first and only argument. 🤷‍♂

Reply:
It works tho?

Reply:
https://github.com/sindresorhus/execa/issues/210